integration_test/third_party_apps_data/applications/postgresql/metadata.yaml

```diff
@@ -31,31 +31,145 @@ minimum_supported_agent_version:
 supported_operating_systems: linux
 supported_app_version: ["10.18+"]
 expected_metrics:
-  - type: workload.googleapis.com/postgresql.backends
+  - kind: GAUGE
+    labels:
+      database: .*
+    monitored_resource: gce_instance
+    type: workload.googleapis.com/postgresql.backends
+    value_type: INT64
+    representative: true
+  - kind: CUMULATIVE
+    monitored_resource: gce_instance
+    type: workload.googleapis.com/postgresql.bgwriter.buffers.allocated
+    value_type: INT64
+  - kind: CUMULATIVE
+    labels:
+      source: backend|backend_fsync|checkpoints|bgwriter
+    monitored_resource: gce_instance
+    type: workload.googleapis.com/postgresql.bgwriter.buffers.writes
+    value_type: INT64
+  - kind: CUMULATIVE
+    labels:
+      type: requested|scheduled
+    monitored_resource: gce_instance
+    type: workload.googleapis.com/postgresql.bgwriter.checkpoint.count
+    value_type: INT64
+  - kind: CUMULATIVE
+    labels:
+      type: sync|write
+    monitored_resource: gce_instance
+    type: workload.googleapis.com/postgresql.bgwriter.duration
     value_type: INT64
-    kind: GAUGE
+  - kind: CUMULATIVE
     monitored_resource: gce_instance
+    type: workload.googleapis.com/postgresql.bgwriter.maxwritten
+    value_type: INT64
+  - kind: CUMULATIVE
     labels:
       database: .*
-    representative: true
-  - type: workload.googleapis.com/postgresql.commits
+      table: .*
+      source: heap_read|heap_hit|idx_read|idx_hit|toast_read|toast_hit|tidx_read|tidx_hit
+    monitored_resource: gce_instance
+    type: workload.googleapis.com/postgresql.blocks_read
+    value_type: INT64
+  - kind: CUMULATIVE
+    labels:
+      database: .*
+    monitored_resource: gce_instance
+    type: workload.googleapis.com/postgresql.commits
     value_type: INT64
-    kind: CUMULATIVE
+  - kind: GAUGE
     monitored_resource: gce_instance
+    type: workload.googleapis.com/postgresql.connection.max
```

Comment:
The scope [doc](https://docs.google.com/document/d/1M9r8CqZl2W9fYz8brzlWntEAqJUC1K43LHGvGkbpiPs/edit) has both `workload.googleapis.com/postgresql.connection.open` and `workload.googleapis.com/postgresql.connection.max`

Is this an intentional change to the scope doc? If so, can we send a suggested edit to the scope doc and have the approver double check it? If not, can we fix the translation so it matches the scope doc?